tests/L0/run_transformer/test_pipeline_parallel_fwd_bwd.py

```diff
@@ -46,23 +70,21 @@ def _forward_backward_test_impl(
         fwd_bwd_func: FwdStepFunc,
         pipeline_model_parallel_world_size: Optional[int],
         virtual_pipeline_model_parallel_size: Optional[int],
+        dtype: Optional[torch.dtype] = torch.float32,
```

Comment:
I'm afraid I don't think this argument will be used so I'd like to either have this test class stick to torch.float32 or leave the `product` of dtypes and "dealloc"
```suggestion
```